react/components/BaseModal.tsx

```diff
@@ -23,15 +23,23 @@ const styles: Record<string, React.CSSProperties> = {
 
 const BaseModal: React.FC<Props> = props => {
   const {
+    open,
     backdrop,
     children,
     container,
-    open = true,
     onBackdropClick,
     keepMounted = false,
     ...rest
   } = props
 
+  useEffect(() => {
+    if (open) {
+      document.body.classList.add('overflow-y-hidden')
+    } else {
+      document.body.classList.remove('overflow-y-hidden')
+    }
```

Comment:
You can use `classList.toggle` with a second conditional argument to add or remove a class: `document.body.classList.toggle('overflow-y-hidden', open)`

Reply:
(thinking it through, it doesn't work on IE 10/11)